controller/src/main/java/org/jboss/as/controller/operations/PathAddressFilter.java

```diff
@@ -33,17 +33,17 @@
 /**
  * @author Emanuel Muckenhuber
  */
-class PathAddressFilter {
+public class PathAddressFilter {
```

Comment:
If this has to be public it must be final now. (Unless it already has subclasses.)